app/src/test/java/com/cmput301w23t09/qrhunter/qrcode/TestQRModel.java

```diff
@@ -2,22 +2,21 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import java.util.ArrayList;
-import java.util.Comparator;
 import org.junit.jupiter.api.Test;
 
 public class TestQRModel {
   // create a mock hash
   private String mockHash() {
     return "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855";
   }
+
   // create a mock qr code
   private QRCode mockCode() {
-    return new QRCode(mockHash(), "mock", null, 10, null, null, null, null);
-  }
-  // create a mock qr code list
-  private ArrayList<QRCode> mockList() {
-    return new ArrayList<QRCode>();
+    QRCode qr = new QRCode();
```

Comment:
If we assume the previous review comment, then these tests are not properly testing the `QRCode`.
In my opinion, I think that since we determine the values of the `mockHash`, we could have `mockCode` run `return new QRCode(mockHash());`

and given that we selected this hash, we would be able to manually calculate it's expected name and score. Allowing for further confirmation that not only are the getter methods returning a value, but that they are returning the proper calculated value.